specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -1045,18 +1394,18 @@
         "name",
         "managedDiskAzureResourceId"
       ],
-      "description": "Lun to expose the ManagedDisk.",
+      "description": "LUN to expose the Azure Managed Disk.",
       "additionalProperties": false,
       "properties": {
         "name": {
           "type": "string",
           "minimum": 7,
           "maximum": 12,
-          "description": "Lun name."
+          "description": "User defined name for iSCSI LUN; example: \"lun0\""
```

Comment:
Please review your property constraints and add them into the description, this way is clear for end users about the constraints they'll face.

Reply:
corrected the constraints - thanks!